Libraries/LibGfx/Bitmap.h

```diff
@@ -79,6 +79,13 @@ static StorageFormat determine_storage_format(BitmapFormat format)
     }
 }
 
+struct BackingStore {
```

Comment:
Since this is not part of the `Bitmap` public API, it would be better to move it into `Bitmap.cpp` to hide it from the world.